rosetta/identifier/operation.go

```diff
@@ -18,5 +18,5 @@ package identifier
 // needed because of the absence Flow does not support sharding.
 type Operation struct {
 	Index        uint `json:"index"`
-	NetworkIndex uint `json:"network_index"`
+	NetworkIndex uint `json:"network_index,omitempty"`
```

Comment:
(Yes the event index starts from 0 as well.)

Reply:
No, the network index has to represent the index that is found on the network. For now, we can indeed use a pointer here, as we have to omit the field if it is empty. However, we should patch it so that we always have the right network index later.